docs/deployment.rst

```diff
@@ -1,6 +1,168 @@
+       arborx/1.2              (D)    globalarrays/5.8      (D)    mfem/4.4.0             (D)    phist/1.9.5                   strumpack/6.3.1-rocm           upcxx/2022.3.0-rocm
+       axom/0.6.1              (D)    hdf5/1.10.7                  mpifileutils/0.11.1    (D)    precice/2.4.0          (D)    strumpack/6.3.1         (D)    upcxx/2022.3.0      (D)
+       butterflypack/2.1.1     (D)    heffte/2.2.0-rocm            nccmp/1.9.0.1          (D)    pumi/2.2.7             (D)    sundials/6.2.0-rocm            veloc/1.5           (D)
+       cabana/0.4.0            (D)    heffte/2.2.0          (D)    nco/5.0.1              (D)    py-cinemasci/1.7.0            sundials/6.2.0          (D)
+       caliper/2.7.0           (D)    hpctoolkit/2022.04.15 (D)    netlib-scalapack/2.2.0 (D)    py-libensemble/0.9.1          superlu-dist/7.2.0-rocm
+       conduit/0.8.3                  hpx/1.7.1-rocm               omega-h/9.34.1         (D)    py-petsc4py/3.17.1            superlu-dist/7.2.0      (D)
+       darshan-runtime/3.3.1   (D)    hpx/1.7.1             (D)    openpmd-api/0.14.4     (D)    py-warpx/22.05-dims2          tasmanian/7.7-rocm
+       datatransferkit/3.1-rc3 (D)    hypre/2.24.0          (D)    papyrus/1.0.2          (D)    py-warpx/22.05-dims3          tasmanian/7.7           (D)
+
+    -------------------------------------------- /gpfs/alpine/csc439/world-shared/E4S/ParaTools/22.05/PrgEnv-gnu/spack/share/spack/lmod/cray-sles15-x86_64/Core --------------------------------------------
+       aml/0.1.0       (D)    charliecloud/0.26             gasnet/2022.3.0       (D)    kokkos/3.6.00-rocm         nrm/0.1.0                  qthreads/1.16      (D)    sz/2.1.12         (D)
+       archer/2.0.0           cray-mpich/8.1.16    (L,D)    ginkgo/1.4.0-rocm            kokkos/3.6.00       (D)    openmpi/4.1.3              raja/0.14.0-rocm          umap/2.1.0        (D)
+       argobots/1.1    (D)    darshan-util/3.3.1   (D)      ginkgo/1.4.0          (D)    legion/21.03.0      (D)    papi/6.0.0.1               raja/0.14.0        (D)    umpire/6.0.0-rocm
+       bolt/2.0        (D)    flit/2.1.0           (D)      gmp/6.2.1             (D)    libunwind/1.6.2     (D)    pdt/3.25.1          (D)    superlu/5.3.0      (D)    umpire/6.0.0      (D)
+       chai/2.4.0-rocm        flux-core/0.38.0     (D)      gotcha/1.0.3          (D)    magma/2.6.2-rocm    (D)    plasma/21.8.29             swig/4.0.2-fortran        zfp/0.5.5         (D)
+       chai/2.4.0      (D)    gasnet/2022.3.0-rocm          kokkos-kernels/3.6.00 (D)    mpark-variant/1.4.0 (D)    py-jupyterhub/1.4.1        swig/4.0.2         (D)
+
+**ParaTools Deployment: E4S 22.05 / PrgEnv-amd**
+
+This deployment is maintained by `ParaTools <https://www.paratools.com/>`_ and may not be officially supported by facility staff.

```

Comment:
i think this sentence is becoming redundant considering we have this for every stack. Perhaps we can just say this once at top of page and just remove all other instances.

Reply:
Aren't we going to eventually cover facility-supported deployments? I.E. won't the 'official' Nersc-controlled Perlmutter 22.05 install also be listed here?

Reply:
well these are already documented in the user facing docs in https://docs.nersc.gov/applications/e4s/ so i think we can have a link from this page to point to our docs. It doesn't make sense to move the content into E4S user docs since we need to actively maintain this documentation and there is a lot of user traffic on https://docs.nersc.gov where user will read the docs

Reply:
If we are only fully documenting Paratools maintained deployments here then yes, that disclaimer should appear just once up top.